backend/btrixcloud/colls.py

```diff
@@ -395,6 +397,23 @@ async def add_successful_crawl_to_collections(self, crawl_id: str, cid: UUID):
             )
             await self.update_crawl_collections(crawl_id)
 
+    async def get_org_public_collections(self, org_slug: str):
+        """List public collections for org"""
+        try:
+            org = await self.orgs.get_org_by_slug(org_slug)
+        # pylint: disable=broad-exception-caught
+        except Exception:
+            # pylint: disable=raise-missing-from
+            raise HTTPException(status_code=404, detail="public_collections_not_found")
+
+        collections, _ = await self.list_collections(
+            org.id, access=CollAccessType.PUBLIC
```

Comment:
> I believe we should be checking if `listPublicCollections` is enabled here, rather than if there's any public collections.

Ah yeah, I missed adding a check for `listPublicCollections` (or its equivalent if we rename) before adding the collections filtered on `CollAccessType.PUBLIC`, will add that.